feat(changelog): iam, em and s3 (#1076)
scaleway/docs-content: changelog/elastic-metal/december2022/2022-12-07-elastic-metal-added-par1.mdx

---
title: Elastic Metal is now available on PAR-1
status: added
author:
    fullname: 'Join the #elastic-metal channel on Slack.'
    url: 'https://slack.scaleway.com/'
date: 2022-12-06
category: compute
product: elastic-metal
---

> A second availability zone (AZ) has been launched for Elastic Metal. This allows you now to build architectures with high-availability and resiliency by getting servers on both the PAR-2 and PAR-1 Availability Zones.

Reviewer: ```suggestion
A second availability zone (AZ) has been launched for Elastic Metal. This allows you to build architectures with high availability and resiliency by creating servers in both the PAR-2 and PAR-1 Availability Zones.
```
